apps/web/src/common/constants/terms.ts

```diff
@@ -4,7 +4,10 @@ interface TermOption {
   label: string
 }
 
+// Reminder: Also edit currentTerm here src/common/utils/getCurrentTerm/index.ts
```

Comment:
Should we move current term here instead?

Reply:
Like maybe define a `currentTerm` constant here, and make `getCurrentTerm` return it

Reply:
done 🙏